src/app/exams/exams-view.component.html

```diff
@@ -12,56 +12,63 @@
+        <ng-container *ngSwitchCase="'grade'">
+          <p><b i18n>Submitted answer:</b></p>
+          <td-markdown [content]="answer?.text || answer"></td-markdown>
+          <mat-radio-group class="planet-radio-group" [(ngModel)]="grade">
+            <mat-radio-button [value]="1" class="planet-radio-button" i18n>Correct</mat-radio-button>
+            <mat-radio-button [value]="0" class="planet-radio-button" i18n>Incorrect</mat-radio-button>
           </mat-radio-group>
-          <div *ngSwitchCase="'selectMultiple'" class="question-list">
-            <mat-checkbox *ngFor="let option of question.choices" [value]="option" (change)="setAnswer($event, option)" [checked]="checkboxState[option.id]">
-              {{option.text}}
-            </mat-checkbox>
-          </div>
+        </ng-container>
+        <ng-container *ngSwitchCase="'view'">
+          <p><b i18n>Response:</b></p>
+          <td-markdown [content]="answer?.text || answer"></td-markdown>
         </ng-container>
       </ng-container>
-      <ng-container *ngSwitchCase="'grade'">
-        <p><b i18n>Submitted answer:</b></p>
-        <td-markdown [content]="answer?.text || answer"></td-markdown>
-        <mat-radio-group class="planet-radio-group" [(ngModel)]="grade">
-          <mat-radio-button [value]="1" class="planet-radio-button" i18n>Correct</mat-radio-button>
-          <mat-radio-button [value]="0" class="planet-radio-button" i18n>Incorrect</mat-radio-button>
-        </mat-radio-group>
-      </ng-container>
-      <ng-container *ngSwitchCase="'view'">
-        <p><b i18n>Response:</b></p>
-        <td-markdown [content]="answer?.text || answer"></td-markdown>
-      </ng-container>
-    </ng-container>
-    <div class="v-align-center">
-      <button
-        *ngIf="mode !== 'view'"
-        mat-raised-button
-        color="primary"
-        (click)="nextQuestion(questionNum)"
-        [planetSubmit]="spinnerOn"
-        [disabled]="answer === undefined || answer === '' || grade === undefined || grade === null"
-        i18n>
-        Submit Answer
-      </button>
-      <span class="small warn-text-color v-align-center" *ngIf="incorrectAnswer">
-        <mat-icon class="margin-lr-3">error</mat-icon><span i18n>Incorrect answer, please try again</span>
-      </span>
+      <div class="v-align-center">
+        <button
+          *ngIf="mode !== 'view'"
+          mat-raised-button
+          color="primary"
+          (click)="nextQuestion(questionNum)"
+          [planetSubmit]="spinnerOn"
+          [disabled]="answer === undefined || answer === '' || grade === undefined || grade === null"
+          i18n>
+          Submit Answer
+        </button>
+        <span class="small warn-text-color v-align-center" *ngIf="incorrectAnswer">
+          <mat-icon class="margin-lr-3">error</mat-icon><span i18n>Incorrect answer, please try again</span>
+        </span>
+      </div>
     </div>
+    <div *ngIf="examType == 'surveys'" class="right-tile">
```

Comment:
We should not have this information showing when a person is taking a survey.

I think showing Submitted by & Submitted on is good for both survey & exam review (`mode === 'grade'), but we can place it in the toolbar next to the question number.